docs/functions.json

```diff
@@ -888,6 +934,25 @@
             }
         ]
     },
+    {
+        "signature": "replace_time(date)",
+        "summary": "Returns the a new date time with the time part replaced by the `time`.",
```

Comment:
I've slept on this and I still think `set_time` would be misleading.. can't think of any example where a set method returns a new value instead of modifying an argument 😬

Reply:
You and your zillion character long expressions. So be it.